stl/inc/xiosbase

```diff
@@ -46,7 +45,6 @@ public:
     static constexpr int hexfloat = 0x3000; // TRANSITION, ABI, GH-3296
 
     static constexpr int boolalpha   = 0x4000;
-    static constexpr int _Stdio      = 0x8000;
     static constexpr int adjustfield = 0x01C0; // left | right | internal
     static constexpr int basefield   = 0x0E00; // dec | oct | hex
     static constexpr int floatfield  = 0x3000; // scientific | fixed

```

Comment:
No change requested, but since we're touching these anyway, could we:
```suggestion
    static constexpr int adjustfield = left | right | internal;
    static constexpr int basefield   = dec | oct | hex;
    static constexpr int floatfield  = scientific | fixed;
```

Reply:
Done. I verified that this compiles before and after:

```
D:\GitHub\STL\out\x64>type meow.cpp
```
```cpp
#include <ios>
using namespace std;

static_assert(ios_base::adjustfield == (ios_base::left | ios_base::right | ios_base::internal));
static_assert(ios_base::basefield == (ios_base::dec | ios_base::oct | ios_base::hex));
static_assert(ios_base::floatfield == (ios_base::scientific | ios_base::fixed));
```
```
D:\GitHub\STL\out\x64>cl /EHsc /nologo /W4 /c meow.cpp
meow.cpp

D:\GitHub\STL\out\x64>
```